src/demo_main.cpp

```diff
@@ -33,27 +35,34 @@ void Demo::Callback(const sensor_msgs::PointCloud2ConstPtr& cloud) {
   pcl::removeNaNFromPointCloud(*pcl_cloud, *pcl_cloud, indices);
 
   pcl::PointIndices::Ptr point_indices(new pcl::PointIndices);
+  PointCloudC::Ptr cropped_cloud(new PointCloudC);
+  sensor_msgs::PointCloud2 msg_out;
+
   pcl::CropBox<PointC> crop;
   crop.setInputCloud(pcl_cloud);
   Eigen::Vector4f min;
-  min << 0, -1, 0.3, 1;
+  min << 0, -0.5, 0.05, 1;
```

Comment:
Would be great if you updated these to be parameters (like the other params below).

Reply:
Completed